osdk/tools/docker/README.md

```diff
@@ -0,0 +1,38 @@
+# OSDK Development Docker Images
+
+OSDK development Docker images are provided to facilitate developing and testing OSDK project. These images can be found in the [asterinas/osdk](https://hub.docker.com/r/asterinas/osdk/) repository on DockerHub.
+
+## Building Docker Images
+
+To build a Docker image for OSDK and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
+
+```bash
+cd <asterinas dir>
+# Build Docker image
+docker buildx build \
+    -f osdk/tools/docker/Dockerfile \
+    --build-arg ASTER_RUST_VERSION=$(grep "channel" rust-toolchain.toml | awk -F '"' '{print $2}') \
+    -t asterinas/osdk:$(cat VERSION)-$(date +%Y%m%d) \
+    .
+```
+
+For the Intel TDX Docker image, it is based on an OSDK Docker image. You can execute the following command:
+
+```bash
+cd <asterinas dir>
+# Build Intel TDX Docker image
+docker buildx build \
+    -f osdk/tools/docker/tdx/Dockerfile \
+    --build-arg ASTER_RUST_VERSION=$(grep "channel" rust-toolchain.toml | awk -F '"' '{print $2}') \
+    --build-arg BASE_VERSION=${BASE_VERSION} \
+    -t asterinas/osdk:$(cat VERSION)-$(date +%Y%m%d)-tdx \
+    .
+```
+
+Here, `BASE_VERSION` represents the version of the OSDK Docker image you want to base it on.
+
+## Tagging and Uploading Docker Images
+
+Regarding the tagging Docker images, please refer to this [link](https://asterinas.github.io/book/to-contribute/version-bump.html).
+
+New versions of OSDK's Docker images are automatically uploaded to DockerHub through Github Actions. Simply submit your PR that updates OSDK's Docker image for review. After getting the project maintainers' approval, the [Docker image building workflow](../../.github/workflows/publish_docker_images.yml) will be started, building the new Docker image and pushing it to DockerHub.
```

Comment:
Most content of this paragraph is duplicated with the one in the version bump documentation, which explains that how version bump triggers Docker image generation. So I think this paragraph can be removed.